ourney/src/main/java/com/lubycon/ourney/domains/expense/entity/ExpenseRepository.java

```diff
@@ -16,4 +18,6 @@
     @Query("SELECT SUM(e.totalPrice) FROM Expense e WHERE e.tripId = :tripId")
     Long getSumByTripId(@Param("tripId") UUID tripId);
 
+    @Query("SELECT DISTINCT new com.lubycon.ourney.domains.expense.dto.ExpensePersonalList(e.expenseId, e.payerId, d.userId, d.price) FROM Expense e INNER JOIN ExpenseDetail d ON d.expenseId = e.expenseId WHERE e.tripId = :tripId")
```

Comment:
쿼리가 길어지면 가독성이 떨어질 수도 있을 것 같아서 이렇게 줄띄움으로 나눠서 보여주는 것도 괜춘할 것 같아용
```suggestion
        @Query("SELECT new com.lubycon.ourney.domains.expense.dto.GetExpenseDetailResponse(U.id, U.nickName, U.profileImg, E.price) " 
        + "FROM ExpenseDetail E " 
        + "INNER JOIN User U ON E.userId = U.id " 
        + "WHERE E.expenseId = :expenseId")
```